src/features/image-map-editor/image-dropzone.tsx

```diff
@@ -0,0 +1,55 @@
+import {
+  useEffect,
+  useCallback,
+} from 'react';
+import { useDropzone } from 'react-dropzone';
+import { useEditor } from './editor.context';
+import * as Styled from './image-dropzone.style';
+
+const dropzoneOptions = {
+  multiple: false,
+  accept: {'image/*': ['.png', '.jpeg', '.jpg', '.svg']}
+};
+
+const ImageDropzone = () => {
+  const {
+    acceptedFiles,
+    getRootProps,
+    getInputProps,
+  } = useDropzone(dropzoneOptions);
+  const { dispatch } = useEditor();
+
+  const addImg = useCallback((file) => {
+    const img = new Image();
+    img.src = URL.createObjectURL(file);
+    img.src = URL.createObjectURL(file);
```

Comment:
Is this line a duplicate one?

Reply:
Ah yes, need to clean up. Thanks for reviewing.